### PR TITLE
Refactor Run intro section for Qiskit Patterns

### LINKED ISSUE #924: [ REFACTOR ] run/introduction to reflect "run on hardware" pattern
[this page](https://docs.quantum.ibm.com/run) should be refactored to explain what the pattern step "Run on hardware" is.

### PR DESCRIPTION
**Pull request by abbycross**
Closes #924

### CODE REVIEW

**Comment by Eric-Arellano** [comment]
Is this change safe to go live to production now, or does it need to wait until the bigger change where we land all of Patterns? I'll be talking in our Monday meeting about the strategy for version control: https://github.com/Qiskit/documentation/issues/996

Tl;dr from content writing perspective:

1. Don't add any changes to existing folders like `start/` that you are not comfortable going live to production
2. Don't add new folders to `open-source-sync-config.json`
3. It is safe to add new folders like `analyze`

**Comment by abbycross** [comment]
I'm assuming this would wait till we have everything in place -- but I will defer to @javabster on that!  Can I merely leave this change unmerged, or should I hide it some other way till we're ready to publish? (I can wait till Monday to hear the gritty details 😆 )

File: docs/run-new/index.mdx
Comment (javabster) [inline docs/run-new/index.mdx:13]:
```suggestion
Step 3 in Qiskit Patterns, "Execute on target hardware," centers around the [Qiskit Runtime primitives](/run/primitives) as the interface with IBM Quantum&trade; hardware, abstracting away error suppression and mitigation techniques so that you can focus more on the outcomes of your quantum computation.
```

Comment (javabster) [reply docs/run-new/index.mdx:13]: I think this is the naming we're going to have to go with

**Review by javabster — APPROVED** [review]
1 small change but otherwise LGTM 🚀

**Pull request merged**
